python/README.md

```diff
@@ -0,0 +1,57 @@
+# Accessing the Cantabular extended API with Python
+
+The API is powered by [GraphQL](https://graphql.org/) which allows users to request only the data that's relevant to them.
+
+The user is presented with a [GraphiQL](https://github.com/graphql/graphiql) session when the API is accessed via a web browser. GraphiQL is the GraphQL integrated development environment (IDE).
+
+Requests to the API via other means return reponses in GraphQL format, which is a little like JSON.
+
+This repository provides some examples on accessing the API with Python.
+
+## Scripts
+
+The scripts demonstrate basic tasks that can be performed using the extended API, but do not exercise the full range of available functionality. They can be used as a starting point for users who want to develop more elaborate applications.
+
+They should be used with `Python 3` and have a dependency on the `requests` library. This should be installed using `pip` if not already present e.g.:
+
+```
+pip3 install requests
+```
+
+### `variables.py`
+
+`variables.py` is used to identify the variables associated with a specific dataset. It constructs a GraphQL query using a user-supplied base URL and dataset name and prints the names of all the available variables. The variables are separated into **rule variables** and other variables. Rule variables are special variables that are used in rule based redaction of categories when cross tabulations are generated.
```

Comment:
`variables.py` prints out all the variables in a dataset that can be used for queries.  The variables are separated...

Reply:
Even then I'm not totally getting away from passive voice. Try again:

`variables.py` prints out all the variables in a dataset available for queries. The program separates the output into **rule variables** and the remainder. A rule variable is one which Cantabular uses for per category publication according to disclosure control rules.

Reply:
OK